features/aanschrijfwijze.feature

```diff
@@ -136,135 +141,622 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
-
-    Voorbeelden: partner heeft een predikaat
-      | aanduidingAanschrijving | samenstelling aanschrijfwijze | geslachtsnaam | voornamen | aanschrijfwijze | regelVoorafgaandAanAanschrijfwijze |
-      | E | VL VV GN          | Berg  | Sjaak          | S. van der Berg | - |
-      | N | VL VV GN-VP GP    | Berg  | Peter          | P. van der Berg-van Hoogh | - |
-      | P | VL VP GP          | Berg  | Marlies        | M. van Hoogh | - |
-      | V | VL VP GP-VV GN    | Berg  | Fleur          | F. van Hoogh-van der Berg | - |
-
-    Voorbeelden: betrokkene heeft een adelijke titel en partner heeft een adellijke titel
-      | aanduidingAanschrijving | samenstelling aanschrijfwijze | geslachtsnaam | voornamen | aanschrijfwijze | regelVoorafgaandAanAanschrijfwijze |
-      | E | VL AT VV GN          | Aedel | Hendrik Willem | H.W. graaf van den Aedel                    | De hooggeboren heer        |
-      | N | VL AT VV GN-AP VP GP | Aedel | Wilhelmina     | W. gravin van den Aedel-barones van Hoogh   | De hooggeboren vrouwe      |
-      | P | VL AP VP GP          | Aedel | Frederique     | F. barones van Hoogh                        | -                          |
-      | V | VL AP VP GP-AT VV GN | Aedel | Emma Louise    | E.L. barones van Hoogh-gravin van den Aedel | De hooggeboren vrouwe      |
-
-  Scenario: meerdere actuele relaties
-    Gegeven de persoon F.C. Groen is getrouwd in 1958 met Geel
-    En de persoon is getrouwd in 1961 met Roodt
-    En geen van beide relaties is beëindigd
-    En de persoon heeft aanduidingAanschrijving='V'
-    Als de persoon wordt geraadpleegd
-    Dan is in het antwoord naam.aanschrijfwijze=F.C. Geel-Groen
-
-  Scenario: meerdere ontbonden relaties gebruikt de laatst ontbonden relatie
-    Gegeven de persoon J. Wit is getrouwd in 1958 met Geel
-    En de persoon is getrouwd in 1961 met Roodt
-    En het huwelijk met Geel is ontbonden in 1960
-    En het huwelijk met Roodt is ontbonden in 2006
-    En de persoon heeft aanduidingAanschrijving='V'
-    Als de persoon wordt geraadpleegd
-    Dan is in het antwoord naam.aanschrijfwijze=J. Roodt-Wit
-
-  Scenario: meerdere ontbonden relaties en oudste relatie is het laatst ontbonden
-    Gegeven de persoon de heer J. Wit is getrouwd in 1958 met Zwart
-    En de persoon is getrouwd in 1961 met Blaauw
-    En het huwelijk met Blaauw is ontbonden in 1983
-    En het huwelijk met Zwart is ontbonden in 2006
-    En de persoon heeft aanduidingAanschrijving='V'
-    Als de persoon wordt geraadpleegd
-    Dan is in het antwoord naam.aanhef=J. Zwart-Wit
+  # In onderstaande tabellen betekenen de afkortingen:
+  # VL = voorletters
+  # VV = voorvoegselGeslachtsnaam
+  # GN = geslachtsnaam
+  # VP = voorvoegselGeslachtsnaam partner
+  # GP = geslachtsnaam partner
+  # AT = adelijke titel
+  # AP = adellijke titel partner (in andersgeslachtelijke vorm)
+  # PK = predikaat
+
+  Rule: De aanschrijfwijze voor een alleenstaande wordt als volgt samengesteld:
```

Comment:
dit hoeft niet waar te zijn. Iemand mag ook aangeven de geslachtsnaam van de ex-partner te gebruiken. Dit gebeurt bijvoorbeeld wanneer de partner overleden is, maar de weduwe wil toch nog met de naam van de overleden echtgenoot aangesproken worden. Zij is dan alleenstaand, maar toch wordt de naam van de (ex)partner gebruikt.
Bepalend is dus niet of de persoon alleenstaand is, bepalend is de aanduidingNaamgebruik

Reply:
deze regel is dus identiek aan wat in de regel v.a. 193 staat, eerste 5 rijen van de tabel (een van de twee is dubbelop)

Reply:
Ik denk dat Melvin met alleenstaand bedoelt: geen huwelijk/partnerschap (ook niet gehad), één van de opties in : [https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/909](url). Dit is natuurlijk een belangrijk scenario.

Reply:
@Melvin, misschien moeten we dit ipv "alleenstaande" het scenario "geen huwelijk partnerschap" noemen.

Reply:
mijn opmerking ging niet over de vraag of dit een belangrijk scenario is om te illustreren, want dat is het zeker. Mijn opmerking gaat over de achterliggende logica (Rule) die beschreven wordt. 
Er is geen andere logica voor een alleenstaande of een persoon met partner. Wanneer je dus deze feature gaat implementeren moet je in de code geen conditie maken op het bestaan van een actuele partner. Deze regel suggereert dat denk ik wel. @MelvLee of gebruik/bedoel jij regels op een andere manier?

Reply:
> @melvin, misschien moeten we dit ipv "alleenstaande" het scenario "geen huwelijk partnerschap" noemen.

Dit is wat ik bedoel. 14 jarigen worden ook door de overheid aangeschreven mbt identificatieplicht. Die zullen volgens mij praktisch nooit een (ontbonden) huwelijk/partnerschap hebben.